spec/amber/dsl/server_spec.cr

```diff
@@ -0,0 +1,57 @@
+require "../../../spec_helper"
+
+module Amber
+  server = Server.new
+  module DSL
+    describe Server do
+
+      describe "pipeline" do
+        context "generating a single ':custom' pipeline" do
+          Amber::Server.configure do |app|
+            pipeline :custom do
+              plug Pipe::Logger.new
+              plug Pipe::Error.new
+            end
+          end
+
+          it "should add ':custom' to the server's pipeline" do
+            pipes = server.settings.handler.pipeline.keys
+            pipes.should contain :custom
+          end
+
+          it "have added pipes in the pipeline" do
+            plugs = server.settings.handler.pipeline[:custom]
+            plugs.first.should be_a Pipe::Logger
+            plugs[1].should be_a Pipe::Error
```

Comment:
I guess this should be 
```crystal
plugs.first?.should be_a Pipe::Logger
plugs[1]?.should be_a Pipe::Error
```

 [.first](https://crystal-lang.org/api/0.23.1/Indexable.html#first-instance-method) may raise an error instead of expectation failure.